backend/app/controllers/users.js

```diff
@@ -268,6 +268,7 @@ module.exports = {
             }
             const existUser = _.first(isExist);
             const userAuthed = yield _createUserOnAuthService(existUser.email, req.body.password, existUser.roleID, req.headers.authorization);
```

Comment:
This line was failing because the auth service now requires passwords to have an upper case, a number etc